src/cli.ts

```diff
@@ -24,6 +24,11 @@ async function main(argv: string[]) {
       "Request headers to redact",
       commaSeparatedList
     )
+    .option(
+      "--prevent-conditional-requests <flag>",
+      "When running in record mode, remove `If-*` headers from outgoing requests in an attempt to prevent the suite of conditional responses being returned (304).",
+      "true"
+    )
```

Comment:
nit: perhaps a negated boolean flag is a nicer CLI interface https://github.com/tj/commander.js/#other-option-types-negatable-boolean-and-booleanvalue? 

`--no-drop-conditional-request-headers` ?

Reply:
Ah sure, nice idea. I didn't bother looking up the docs for this command line arg parsing library to see if there was an option for that.